### ISSUE TITLE
Support to Stream audio from selected sound output only.

### ISSUE THREAD

**Issue by miXza81** [acidhax/chromecast-audio-stream#98]
Hi.

Support to Stream audio from selected sound output/card (S/PDIF, HDMI, line-out...) only would be nice feature.

**Comment by FilipChalupa** [comment]
Having Chromecast as another audio output device sounds even better to me, leaving the sound duplication across multiple devices to OS.

![image](https://user-images.githubusercontent.com/1045362/33241243-10b0450c-d2c3-11e7-84cc-69a4c4f6315a.png)

**Comment by kuhlmannb** [comment]
Agreed. I'm able to connect to my Google Home, but music still plays through the computer speakers, which are tinny and therefore it defeats the purpose for me. Thanks!
